gpii/node_modules/testing/src/Fixtures.js

```diff
@@ -190,8 +190,8 @@ fluid.defaults("gpii.test.couchEnvironment", {
                 "%gpii-universal/tests/data/dbData/gpiiKeys.json",
                 "%gpii-universal/build/tests/dbData/gpiiKeys.json",
                 "%gpii-universal/build/tests/dbData/prefsSafes.json",
-                "%gpii-universal/build/dbData/user/gpiiKeys.json",
-                "%gpii-universal/build/dbData/user/prefsSafes.json",
+                "%gpii-universal/build/dbData/snapset/gpiiKeys.json",
```

Comment:
Great to discover the data in "dbData/user/" is not used. Please also remove the generation of this directory from [package.json](https://github.com/GPII/universal/blob/master/package.json#L79) and references of these directory from READMEs. Thanks.

Reply:
I'm more conservative than you.  What I know is that the `dbData/user/` keys and prefsSafes are not used in universal testing:  not in the browser tests, the node tests, nor the production tests.

But, I don't know beyond that.  They could be used elsewhere, say by something in the linux or windows repository, or some other code elsewhere.  I'll ask on the architecture list to see if anyone out there needs them, just in case.